indico/htdocs/js/utils/forms.js

```diff
@@ -36,5 +36,12 @@
             var passwordField = $(this.form).find('input[name="' + confirmField.data('confirmPassword') + '"]');
             validatePasswordConfirmation(passwordField, confirmField);
         });
+
+        $('form [data-disabled-until-change]').closest('form').each(function() {
+            $(this).data('serialized', $(this).serialize());
+        }).on('change input', function() {
+            var changed = $(this).serialize() == $(this).data('serialized');
```

Comment:
that variable is true if there are no changes, so "changed" is not the best name...